src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
 // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const a = wall.clientWidth;
+const b = wall.clientHeight;
+
+spider.style.top = `${(b - spider.height) / 2}px`;
+spider.style.left = `${(a - spider.width) / 2}px`;
```

Comment:
Ensure that 'spider.height' and 'spider.width' are correctly retrieving the dimensions of the 'spider' element. If these properties are not defined, consider using 'spider.offsetHeight' and 'spider.offsetWidth' instead to get the actual rendered dimensions.